2020-05-14 02:00:44,450 | sandbox | INFO | Some Random log
2020-05-14 03:30:54,728 | evaluator | INFO | Submit 42 | Downloading file ABProblem.01.in with hash 43586469c2fa5c8045fea3143acb5cfc from URL: http://localhost/data/problems/ABProblem/Tests/ABProblem.01.in
2020-05-14 03:30:54,732 | network | ERROR | Could not complete request to http://localhost/data/problems/ABProblem/Tests/ABProblem.01.in: got exception HTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /data/problems/ABProblem/Tests/ABProblem.01.in (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f26d37b4850>: Failed to establish a new connection: [Errno 111] Connection refused'))
2020-05-14 03:30:54,732 | evaluator | ERROR | Submit 42 | Could not download tests. Exception was: 'NoneType' object has no attribute 'status_code'
2020-05-14 03:31:39,774 | evaluator | INFO | Submit 42 | Downloading file ABProblem.01.in with hash 43586469c2fa5c8045fea3143acb5cfc from URL: http://localhost/data/problems/ABProblem/Tests/ABProblem.01.in
2020-05-14 03:31:39,778 | network | ERROR | Could not complete request to http://localhost/data/problems/ABProblem/Tests/ABProblem.01.in: got exception HTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /data/problems/ABProblem/Tests/ABProblem.01.in (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f2e39cae8b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2020-05-14 03:31:39,779 | evaluator | ERROR | Submit 42 | Could not download tests. Exception was: 'NoneType' object has no attribute 'status_code'
2020-05-14 03:32:04,143 | evaluator | INFO | Submit 42 | Downloading file ABProblem.01.in with hash 43586469c2fa5c8045fea3143acb5cfc from URL: http://localhost/data/problems/ABProblem/Tests/ABProblem.01.in
2020-05-14 03:32:04,148 | network | ERROR | Could not complete request to http://localhost/data/problems/ABProblem/Tests/ABProblem.01.in: got exception HTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /data/problems/ABProblem/Tests/ABProblem.01.in (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe897f4c8b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2020-05-14 03:32:04,148 | evaluator | ERROR | Submit 42 | Could not download tests. Exception was: 'NoneType' object has no attribute 'status_code'
2020-05-14 03:32:24,193 | evaluator | INFO | Submit 42 | Downloading file ABProblem.01.in with hash 43586469c2fa5c8045fea3143acb5cfc from URL: http://localhost/data/problems/ABProblem/Tests/ABProblem.01.in
2020-05-14 03:32:24,198 | evaluator | INFO | Submit 42 | Downloading file ABProblem.01.sol with hash ed265bc903a5a097f61d3ec064d96d2e from URL: http://localhost/data/problems/ABProblem/Tests/ABProblem.01.sol
2020-05-14 03:32:24,203 | evaluator | INFO | Submit 42 | Downloading file ABProblem.02.in with hash 75e0f7cbc9a2e1c6126e256773877ec3 from URL: http://localhost/data/problems/ABProblem/Tests/ABProblem.02.in
2020-05-14 03:32:24,207 | evaluator | INFO | Submit 42 | Downloading file ABProblem.02.sol with hash 6d40ec74b08c42d003e9562fe524e025 from URL: http://localhost/data/problems/ABProblem/Tests/ABProblem.02.sol
2020-05-14 03:32:24,213 | evaluator | INFO | Submit 42 | Downloading file ABProblem.03.in with hash dc361c167527c45e3e7a9063b638e976 from URL: http://localhost/data/problems/ABProblem/Tests/ABProblem.03.in
2020-05-14 03:32:24,218 | evaluator | INFO | Submit 42 | Downloading file ABProblem.03.sol with hash 7ae40fa8d78f21b07697d4c2d8b7bdba from URL: http://localhost/data/problems/ABProblem/Tests/ABProblem.03.sol
2020-05-15 12:53:03,411 | initializer | INFO | Initializing...
2020-05-15 12:53:03,411 | executors | INFO | Setting up executor01...
2020-05-15 12:53:03,418 | executors | INFO |   >> cleaning sandbox directory...
2020-05-15 12:53:03,493 | executors | INFO |   >> creating sandbox directory...
2020-05-15 12:53:03,493 | executors | INFO |   >> mounting system paths...
2020-05-15 12:53:03,545 | executors | INFO |   >> creating home directory...
2020-05-15 12:53:03,546 | executors | INFO |   >> completed!
2020-05-15 12:53:03,546 | executors | INFO | Setting up executor02...
2020-05-15 12:53:03,553 | executors | INFO |   >> cleaning sandbox directory...
2020-05-15 12:53:03,628 | executors | INFO |   >> creating sandbox directory...
2020-05-15 12:53:03,629 | executors | INFO |   >> mounting system paths...
2020-05-15 12:53:03,681 | executors | INFO |   >> creating home directory...
2020-05-15 12:53:03,682 | executors | INFO |   >> completed!
2020-05-15 12:53:03,682 | executors | INFO | Setting up executor03...
2020-05-15 12:53:03,688 | executors | INFO |   >> cleaning sandbox directory...
2020-05-15 12:53:03,762 | executors | INFO |   >> creating sandbox directory...
2020-05-15 12:53:03,763 | executors | INFO |   >> mounting system paths...
2020-05-15 12:53:03,815 | executors | INFO |   >> creating home directory...
2020-05-15 12:53:03,816 | executors | INFO |   >> completed!
2020-05-15 12:53:03,816 | executors | INFO | Executors set up successfully.
2020-05-15 12:53:03,832 | executors | INFO | Executors queued successfully.
2020-05-15 12:53:03,832 | sandbox | INFO | Sandbox initialized successfully.
2020-05-15 12:53:03,832 | initializer | INFO | Initialization completed successfully!
